themes/default/content/resources/azure-github-workshop/index.md

```diff
@@ -1,8 +1,8 @@
 ---
 # Name of the webinar.
-title: "Getting Started with Azure IaC and Review Stacks"
-title_tag: "Getting Started with Azure IaC and Review Stacks (July 2023 Workshop)"
-meta_desc: "In this workshop, we’ll explore the new Azure Native v2 provider and Pulumi Review Stacks on GitHub"
+title: "Getting Started with Azure IaC and Pulumi Deployments"
+title_tag: "Getting Started with Azure IaC and Pulumi Deployments"
+meta_desc: "In this workshop, we’ll explore the new Azure Native v2 provider and how Pulumi Deployments make it easier to validate both infrastructure and app code before release."
```

Comment:
> Hey Sara, It looks like it is failing because the meta description is too long. I think it needs to be less than 160 characters.

Thanks! How can you tell that was it? I seen code 2 before and it was a space at the end of a line.

Reply:
you can pretty much ignore the error code since it usually says code 2 no matter the error. A few lines up it has the message here :slightly_smiling_face: : https://github.com/pulumi/pulumi-hugo/actions/runs/6314148641/job/17143767381#step:7:53